cdbot/cogs/fun.py

```diff
@@ -165,26 +165,32 @@ def check(reaction, user):
                     await message.delete()
             finally:
                 await sent.delete()
+                
+        if message.content.startswith(":react ") or message.content.startswith(":emojify "):
+            # Don't react to invocations of :react
+            return
 
         # React if a message contains an @here or @everyone mention.
         if any(mention in message.content for mention in ("@here", "@everyone")):
             await message.add_reaction("\N{SLIGHTLY FROWNING FACE}")
-            await emojify(message, "who pinged")
+            return await emojify(message, "who pinged")
 
         # React FBI OPEN UP if message contains trigger words.
         triggers = ["child", "fbi", "loli", "hentai", "illegal", "maltego"]
         if any(trigger in message.content.lower() for trigger in triggers):
-            await emojify(message, "fbi open up")
+            return await emojify(message, "fbi open up")
```

Comment:
Would it be worth moving this into the collection of other reactions on lines 31-36?

Reply:
Yeah, I guess, just wondering if it would be nice having so many duplicate responses in the dictionary. I don't think it's a huge issue though, will move after adding the ratelimit .